action.yml

```diff
@@ -145,23 +150,27 @@ runs:
         TAG: ${{ inputs.tag }}
         RELEASE_DESCRIPTION: ${{ inputs.release_description }}
         GITHUB_REPOSITORY: ${{ github.repository }}
+        GITHUB_SHA: ${{ github.sha }}
       run: |
-        # Create new release via GitHub API
-        # https://docs.github.com/en/rest/releases/releases#create-a-release
         # Escape double quotes '"' => '\"'
         RELEASE_DESCRIPTION=${RELEASE_DESCRIPTION//\"/\\\"}
-        RESP_CODE=$(curl -w %{http_code} -s -o /dev/null \
+        # Create new release via GitHub API
+        # https://docs.github.com/en/rest/releases/releases#create-a-release
+        # TODO: change __response.json back to /dev/null
```

Comment:
TODO comment here - I'm guessing something you intended to do before marking ready for review?

Reply:
Removed